tests/util/governance.ts

```diff
@@ -0,0 +1,76 @@
+import { Keyring } from "@polkadot/api";
+import { AddressOrPair, ApiTypes, SubmittableExtrinsic } from "@polkadot/api/types";
+import {
+  ALITH_PRIV_KEY,
+  BALTATHAR_PRIV_KEY,
+  CHARLETH_PRIV_KEY,
+  DOROTHY_PRIV_KEY,
+} from "./constants";
+import { DevTestContext } from "./setup-dev-tests";
+import { createBlockWithExtrinsic } from "./substrate-rpc";
+
+const keyring = new Keyring({ type: "ethereum" });
+
+export const execFromTwoThirdsOfCouncil = async <
+  Call extends SubmittableExtrinsic<ApiType>,
+  ApiType extends ApiTypes
+>(
+  context: DevTestContext,
+  polkadotCall: Call
+) => {
+  // Council members
+  const charleth = await keyring.addFromUri(CHARLETH_PRIV_KEY, null, "ethereum");
+  const dorothy = await keyring.addFromUri(DOROTHY_PRIV_KEY, null, "ethereum");
+
+  // Charleth submit the proposal to the council (and therefore implicitly votes for)
+  let lengthBound = 1_000;
```

Comment:
Can we gather the proposal length and just use that number? 1000 sounds a bit arbitrary, what if for whatever reason one day we use this function for a proposal whose length is bigger?

Reply:
Done